### ISSUE TITLE
How to set the weights of model.ckpt.data to random for custom dataset

### ISSUE THREAD

**Issue by Petros626** [tensorflow/models#10728]
Hello Guys,

I use the `ssd_mobilenet_v2_quantized_300x300_coco` for traing on bird eye view images of LiDAR data. The sparse data aren't good enough for training, even fine tuning the parameter like **learning rate, change optimizer** helped.

Is there a way to open the file `model.ckpt.data-00000-of-00001` and set its weights to **random** or **reset** to reload the **new** file in the `config file` for training the model again?

Please help me with detailed instructions I need it for my degree...

**Comment by seboz123** [comment]
Just remove: 
`fine_tune_checkpoint_type: "detection" `
`fine_tune_checkpoint_version: V2`
`fine_tune_checkpoint: "/data/pretrained_model/ckpt-0"`
  from your pipeline config file

**Comment by Petros626** [comment]
> Just remove: `fine_tune_checkpoint_type: "detection" ` `fine_tune_checkpoint_version: V2` `fine_tune_checkpoint: "/data/pretrained_model/ckpt-0"` from your pipeline config file

Hey @seboz123 ,

thank you for the quick response.

The second message parameter is unknown for me, but the other two are in the `config file`:

` fine_tune_checkpoint: "PATH_TO_BE_CONFIGURED/model.ckpt"`
 ` fine_tune_checkpoint_type:  "detection"`

I've read some approaches, where people did this, but the opinion was divergent and official authors never confirm this. 
So there's no need to load the `model.ckpt.data-00000-of-00001` in an IDE and set it there, just to be 100% sure?

**Comment by sushreebarsa** [comment]
@Petros626 
In order to expedite the trouble-shooting process, could you please provide the entire URL of the repository which you are using. Please provide more details on the issue reported here.Thank you!

**Comment by Petros626** [comment]
> @Petros626 In order to expedite the trouble-shooting process, could you please provide the entire URL of the repository which you are using. Please provide more details on the issue reported here.Thank you!

Hey @sushreebarsa,

in my case I'm using the TensorFlow 1 Object Detection API (https://github.com/tensorflow/models/tree/master/research/object_detection) and requested to reset all the pretrained weights, so that you could train the model architecture from scratch with your dataset and the parameters, which the authors figured out.
